js/src/views/ParityBar/parityBar.js

```diff
@@ -29,7 +29,8 @@ import styles from './parityBar.css';
 class ParityBar extends Component {
   static propTypes = {
     pending: PropTypes.array,
-    dapp: PropTypes.bool
+    dapp: PropTypes.bool,
```

Comment:
Bad example since this was not done, but we do prefer to re-arrange alphabetically when touching (have not done a bulk-re-arrange, but when touching we do do it.)